requirements.txt

```diff
@@ -1,5 +1,6 @@
 compas @ git+https://github.com/compas-dev/compas@main
```

Comment:
change this to: compas >= 2.0.0b2

Reply:
`NetworkObject` disappeared in the latest compas. any clue? :)